backend/core/tests/test_trusted_users.py

```diff
@@ -0,0 +1,25 @@
+from django.test import TestCase
+
+from core.models import User, EmailDomain
+
+
+class TestTrustedUsers(TestCase):
+    def test_user_trusted_based_on_email_domain(self):
+        user = User.objects.create_user(username="test-user", email="test@example.test")
+        email_domain = EmailDomain.objects.filter(domain="@example.test").first()
+        self.assertIsNotNone(email_domain)
+        self.assertFalse(user.is_trusted)
+
+        email_domain.status = EmailDomain.STATUS_ACCEPTED
+        email_domain.save()
```

Comment:
```suggestion
        email_domain.update(status=EmailDomain.STATUS_ACCEPTED)
```

Reply:
`.update()` can be used on a `QuerySet` but not on a model instance, it seems.